proto/lighthouse-result.proto

```diff
@@ -6,42 +6,41 @@ import "google/protobuf/wrappers.proto";
 
 // Canonical list of errors created by Lighthouse.
 enum LighthouseError {
-  UNDEFINED = 0;
   // No error in Lighthouse; the results are reliable.
-  NO_ERROR = 1;
+  NO_ERROR = 0;
```

Comment:
This is kindof a bummer, we can never undo this it seems, so we're 100% happy with it right?

Reply:
or rather, do they all need to be sequential? It'd be nice if is there a way for 0 and 1 to both be no error somehow or at least not impact every single other type

Reply:
They don't have to be sequential at all, we can reserve as much space as we want for anything.  But why would we need to?  I don't follow.

We can also set up `alias`ing for enums.  Like `NO_ERROR = 0;` and `UNDEFINED=0;` and share fieldnumbers for multiple inputs.

Reply:
https://developers.google.com/protocol-buffers/docs/proto3#assigning-field-numbers

Reply:
I just mean that all consumers of the proto would find their values shifted after this change right? We don't have any consumers yet, and I'm sure you've already thought this through :) but I wanted to know what the plan would be if we needed to make another change to the enums like this, all the values wouldn't change right we'd use one of those handy aliases or something?

Reply:
Yes.  If we had any consumers they'd all be off by one.  Once we start to use this proto we can't change it.  `NO_ERROR` will always be the default, but we can alias more defaults and add new ones and mark numbers as resolved.